### PR TITLE
Test PR

### PR DESCRIPTION
**Pull request by lydiapuric**
Please ensure your pull request adheres to the following guidelines:
- [ ] make sure to link the related issues in this description
- [ ] when merging / squashing, make sure the fixed issue references are visible in the commits, for easy compilation of release notes

## Related Issues

Thanks for contributing!

### CODE REVIEW

File: src/queries/sc/domain-hints.sql [hunk 1]
```diff
@@ -3,7 +3,7 @@
 --- Cache-Control: max-age=3600
 --- timezone: UTC
 --- device: all
---- domainkey: secret
+--- domain key: secret
```
Comment (dbrrt) [inline src/queries/sc/domain-hints.sql:6]: There's no space between `domain` and `key`

Comment (lydiapuric) [reply src/queries/sc/domain-hints.sql:6]: Correct @dbrrt, this was only a test

**Comment by trieloff** [comment]
@lydiapuric you now have write permissions on the repo

**Comment by lydiapuric** [comment]
@trieloff Thanks a lot!

**Pull request closed without merging**